### PR TITLE
Collapsible sections styling

### PR DESCRIPTION
**Pull request by nickcall**
- Adds styles for collapsible sections (within the article body) when the user is on small screen.

### CODE REVIEW

File: packages/nature-article/scss/50-components/article-enhanced.scss [hunk 1]
```diff
@@ -181,23 +181,73 @@ when source text is already uppercase, hence the use of font-size, text-transfor
...
+
+			&:after {
+				transform: translateY(-50%);
+				transition-duration: .2s;
+				-webkit-transform: translateY(-50%);
+				-webkit-transition-duration: .2s;
+			}
+		}
+
+		&--collapible {
```
Comment (benjclark) [inline packages/nature-article/scss/50-components/article-enhanced.scss]: should be `collapsible` ?

File: packages/nature-article/scss/50-components/article-enhanced.scss [hunk 2]
```diff
@@ -181,23 +181,73 @@ when source text is already uppercase, hence the use of font-size, text-transfor
...
+			transition-duration: .2s;
+			position: absolute;
+			top: 50%;
+			right: 25px;
+		}
+
+		&--closed {
+			overflow: hidden;
+			text-overflow: ellipsis;
+			-ms-text-overflow: ellipsis;
```
Comment (benjclark) [inline packages/nature-article/scss/50-components/article-enhanced.scss]: Are you using gulp to compile your css? If so, the sites-nature gulpfile has autoprefixer, so you might find that you don't need to prefix your styles anymore

File: packages/nature-article/scss/50-components/article-enhanced.scss (hunk 1 above)
Comment (nickcall) [reply packages/nature-article/scss/50-components/article-enhanced.scss]: Actually looking into that style I don't think it's used so will remove.

File: packages/nature-article/scss/50-components/article-enhanced.scss (hunk 2 above)
Comment (nickcall) [reply packages/nature-article/scss/50-components/article-enhanced.scss]: thanks for the help Ben, next commit will remove all those styles

**Pull request merged**